assets/agenda/components/AgendaItemDetails.jsx

```diff
@@ -50,10 +54,8 @@ export default function AgendaItemDetails({item, user, actions, onClose}) {
                 </ContentBar>
             </ContentHeader>
 
-            <Article image={map} covering={hasCoverages(item)}>
+            <Article covering={hasCoverages(item)} header={header}>
```

Comment:
might be cleaner to pass there item and handle the map name and time in article component